### DISCUSSION TITLE
5월 18일 백앤드 회의

### DISCUSSION THREAD

**Discussion by joohongpark** [PPiing/PPoong#36] [Backend discussions]
## 게임
게임은 비교적 고려해야 할 사항이 많으므로 잠시 미뤄두도록 결정
## 해야 할 일
- socket.io 예제 문서와 nest.js 예제 문서를 보면서 채팅 구현에 대한 학습
- 중간중간 디스코드에서 의견 공유 많이 하기
## 다음 회의
- 5월 22일 (일) 밤 8시

**Comment by joohongpark** [comment]
@joohongpark @yamkim @subin195-09 @jinbekim

**Reply by jinbekim** [reply]
게임은 그러면 채팅 하고 다시 이야기 하나요?

**Reply by joohongpark** [reply]
음 어제 게임 관련하여 이야기를 계속 나누어 봤었는데 게임도 결국 HTTP API, 소켓 통신을 사용하지만 기술적 난이도가 있다고 생각하여 채팅 기능부터 구현해보자 하는 쪽으로 이야기가 나왔었어요. 관련 레퍼런스도 많고 미리 구현해보며 관련 지식을 쉽게 체득할 수 있다고 생각해 채팅을 구현하여 관련 기술에 어느정도 숙달될 때 게임을 다시 손대는게 어떨까 하고 의견이 나왔었습니다. 어떻게 생각하시나요?

**Reply by jinbekim** [reply]
그럼 저는 일단 일요일까지 게임쪽 공부 해봐도 될까요?

**Reply by joohongpark** [reply]
넵넵 좋아여
